packages/web3/src/index.ts

```diff
@@ -61,6 +61,9 @@ export class Web3 extends Web3Context<EthExecutionAPI> {
 		};
 	};
 
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	private readonly _contracts: Contract<any>[] = [];
```

Comment:
do we ever need to remove contracts?

Reply:
As the instance of the `Web3` object will be garbage collected, these contracts will be removed.